src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtilsUnitTest.java

```diff
@@ -1,12 +1,16 @@
 package org.broadinstitute.hellbender.tools.funcotator.dataSources;
 
+import org.aeonbits.owner.util.Collections;
```

Comment:
Did you intend to import this class, or did you mean to import `java.util.Collections`?

Reply:
Wow.  No - IntelliJ got a little funny with this one I guess.